src/test/codewhisperer/commands/onAcceptance.test.ts

```diff
@@ -24,6 +24,7 @@ describe('onAcceptance', function () {
     describe('onAcceptance', function () {
         beforeEach(async function () {
             await resetCodeWhispererGlobalVariables()
+            session.reset()
```

Comment:
@justinmk3 I suspect the previous fix PR doesn't work as expected because of async/sync things. Test case itself is async, beforeEach is async and afterEach is sync, any guidance for this thanks!

Reply:
mocha awaits async `beforeEach`. You can verify this by adding `await sleep(10000)` here. It won't run any `it()` until the sleep() has completed.

Reply:
understood, so by saying `mocha awaits async beforeEach/afterEach/etc.`, we can say L31 itself should already do its work correctly and clean the states after each test case, and the root cause for test failure might be something else.

Reply:
gotcha, let's add one in beforeEach() then